### PR TITLE
Small RST tweaks to docstrings

### PR DESCRIPTION
**Pull request by peterjc**
Spiritual continuation of work in #9820, some minor RST issues noticed and fixed while testing https://github.com/peterjc/flake8-rst-docstrings/pull/27

### CODE REVIEW

File: astropy/time/tests/test_methods.py [hunk 1]
```diff
@@ -317,9 +317,9 @@ def setup(self):
     @pytest.mark.parametrize('kw, func', itertools.product(kwargs, functions))
     def test_argfuncs(self, kw, func, masked):
         """
-        Test that np.argfunc(jd, **kw) is the same as t0.argfunc(**kw) where
-        jd is a similarly shaped array with the same ordinal properties but
-        all integer values.  Also test the same for t1 which has the same
+        Test that ``np.argfunc(jd, **kw)`` is the same as ``t0.argfunc(**kw)``
+        where jd is a similarly shaped array with the same ordinal properties
```
Comment (pllim) [inline astropy/time/tests/test_methods.py:321]: To be consistent?
```suggestion
        where ``jd`` is a similarly shaped array with the same ordinal properties
```

**Comment by bsipocz** [comment]
Thanks @peterjc!

File: astropy/time/tests/test_methods.py (hunk 1 above)
Comment (peterjc) [reply astropy/time/tests/test_methods.py]: I nearly made that edit myself.

**Comment by peterjc** [comment]
Happy to contribute.

**Pull request merged**